src/target/adiv5.c

```diff
@@ -128,9 +134,9 @@ enum arm_arch {
 };
 
 #ifdef ENABLE_DEBUG
```

Comment:
It's not for this PR, but at some point we should make this depend on ADIv5_DEBUG rather than the general debug define, so we can enable and disable detailed ADIv5 decoder logic debug separately in the menuconfig in v2.

Reply:
noted, that sounds like a good idea